package.json

```diff
@@ -63,7 +63,8 @@
 			"jsdoc/require-returns": "off",
 			"jsdoc/require-param-description": "off",
 			"jsdoc/no-undefined-types": "off",
-			"jsdoc/require-returns-description": "off"
+			"jsdoc/require-returns-description": "off",
+      "no-unused-vars": "off"
```

Comment:
Added this rule -- could remove it but we'd have to update a bunch of javascript that I'm not very comfortable changing. What do you think @greatislander?

Reply:
I think it would be good to review most of the affected code as I wrote most of it 5 years ago and don't trust it anymore? But I think you're fine to add that rule exemption. The one case where I would make a code change is here:

https://github.com/pressbooks/pressbooks/blob/0d278b1b7613b74052ede54fe6ee0a16a6512a44/assets/src/scripts/contributors.js#L2-L10

You'll note that the whole file is wrapped in `jQuery( function ( $ ) ) { ... });`. However, none of the code in the file uses `$()` to call jQuery methods. I think it would make sense to update that file so that everything inside the `function ( $ ) {}` actually uses the dollar sign for jQuery.

Reply:
Created new issue to address: #2924